CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Full list of differences found in [this compare.](https://github.com/open-teleme
 * :stop_sign: [DEPRECATION] Deprecate IntSum, IntGauge, and IntDataPoint (#278)
 * :stop_sign: [DEPRECATION] Deprecate IntExemplar (#281)
 * :stop_sign: [DEPRECATION] Deprecate IntHistogram (#270)
+* :stop_sign: [BREAKING] Deprecate `labels` field from NumberDataPoint, HistogramDataPoint, SummaryDataPoint and add equivalent `attributes` field (#283)
```

Comment:
Discussed in Data Model SiG:  This isn't a breaking change because the deprecated fields are still included in the protocol.   Once removed, that would be breaking.